src/routes/player/WaitingForOtherPlayers.tsx

```diff
@@ -0,0 +1,83 @@
+import { socket } from "../../socket";
+import { Link } from "react-router-dom";
+import { observable } from "@legendapp/state";
+import { WS_EVENTS } from "../../socket/events";
+import { useObserveEffect } from "@legendapp/state/react"
+import { persistObservable } from "@legendapp/state/persist"
+import { enableReactTracking } from "@legendapp/state/config/enableReactTracking"
+import { ObservablePersistLocalStorage } from '@legendapp/state/persist-plugins/local-storage'
+
+// ShowAnswer not implemented yet (accepts bool representing correct/not)
+import ShowAnswer from "./ShowAnswer"
+
+const state$ = observable({
+  player: {
+    id: '',
+  },
+  isAnswerCorrect: false,
+  waitingForOthers: true, 
+  showAnswerPage: false, 
+  correctAnswer: '',
+  selectedAnswer: '',
+});
+
+function WaitingForOtherPlayersPage() {
+  return (
+    <>
+      <div className="bg-gray-100 min-h-screen flex flex-col items-center gap-5">
+        <div className="bg-purple-800 min-w-full p-5">
+          <Link to="/" className="text-6xl text-white font-bold">
+            !Toohak
+          </Link>
+        </div>
+        <div className="flex gap-3">
+          <>
+            <h1 className="test-3x1 font-bold underline">Waiting for other players to answer...</h1>
+          </>
+        </div>
+      </div>
+    </>
+  )
+}
+
+function WaitingForOtherPlayers() {
+
+  enableReactTracking({ auto: true });
+  persistObservable(state$, {
```

Comment:
why do we need use this?

Reply:
I don't believe it is necessary, removed